easybuild/framework/easyblock.py

```diff
@@ -1054,10 +1054,9 @@ def make_module_extra_extensions(self):
         lines = [self.module_extra_extensions]
```

Comment:
Yes, this is extra stuff that can go into the module file that is specific to a particular extension, which is returned by the `run` method.
This is still broken now, since every call to `run` is being skipped under `--module-only`...

It's possible to fix this, but that's orthogonal to this fix imho. I'll mention this in #1877, which is sort of a tracking issue for problems with `--module-only`